src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/DeviceInfo/DeviceInfo.android.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Java.Util;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class DeviceInfo
+	{
+		static string CurrentCultureName => Locale.Default.Language;
```

Comment:
Localization requires not only the language but also the region

Reply:
I initially tried using `Locale.Default.ToString().Replace('_', '-')` to get the region as well, but this was error prone. Not all android language-region pairs are supported by C#.

So I'm returning language only to definitely get valid `CultureInfo`.

Do people have different localization values for same language, but different regions?

Reply:
Yes my app uses different region settings for Arabic language. The numbers, calendar, date and time differ in different regions

Reply:
@maxkoshevoi  Even with 'en' language, both date and time formatting vary between regions.
As this returns a string, I think it should be left to the programmer to ensure that the culture is supported by .Net and what fallback they want to take.
It might be that the developer has defined custom cultures to match the android ones etc.